CHANGELOG.adoc

```diff
@@ -5,6 +5,24 @@
 
 == DFX
 
+=== feat: Configure subnet type of local replica
+
+The local replica sets its parameters according to the subnet type defined in defaults.replica.subnet_type, defaulting to 'application' when none is specified.
+This makes it less likely to accidentally hit the 'cycles limit exceeded' error in production.
```

Comment:
```suggestion
This makes it less likely to accidentally hit the 'cycles limit exceeded' error in production.  Since the previous default was `system`, you may see these types errors in development instead.
```